src/scripts/syncBreaches.js

```diff
@@ -129,14 +134,13 @@ if (seen.size !== breaches.length) {
 }
 
 await getBreachIcons(breaches)
+
 Sentry.captureCheckIn({
   checkInId,
   monitorSlug: SENTRY_SLUG,
   status: "ok"
-});
-process.exit()
-
-
+})
+setTimeout(process.exit, 1000)
```

Comment:
Hacky, but we need the sentry capture to execute prior to process.exit, otherwise we have check-ins that are forever pending